src/app/components/WatchList/WatchList.stories.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import WatchListHeader from './WatchListHeader';
+
+export default {
+  title: 'Component/WatchList',
+  component: WatchListHeader,
+};
+
+export const WatchList = (): JSX.Element => (
+  <WatchListHeader headline="Deine Merkliste" />
+);
```

Comment:
I am confused, is it `WatchList` or `WatchListHeader`? The Header component is a `div`, but I would expect a `header` element based on the name.

Reply:
It should be header, I will rename it